server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -362,12 +362,7 @@ private void completeFacets(Facets facets, SearchWsRequest request, Request wsRe
     addMandatoryValuesToFacet(facets, LANGUAGES, request.getLanguages());
     addMandatoryValuesToFacet(facets, TAGS, request.getTags());
     addMandatoryValuesToFacet(facets, TYPES, RuleType.names());
-    List<String> actionPlans = Lists.newArrayList("");
-    List<String> actionPlansFromRequest = request.getActionPlans();
-    if (actionPlansFromRequest != null) {
-      actionPlans.addAll(actionPlansFromRequest);
-    }
-    addMandatoryValuesToFacet(facets, DEPRECATED_ACTION_PLANS, actionPlans);
+    addMandatoryValuesToFacet(facets, DEPRECATED_ACTION_PLANS, Collections.<String>emptyList());
```

Comment:
I keep them in SearchWsRequest for the same reason I keep them in the web api (depreciation policy). The xxxWsRequest are a 1 to 1 equivalent of WS. When the web api changes, the WsRequest should change.
Otherwise!